### DISCUSSION TITLE
how to stream original images in the FaceDetection demo

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#773] [Q&A] [answered]
Hi, in the FaceDetection demo the output is streamed via wifi and can be seen using `opencv-viewer.py`. I want to stream the original images captured by the camera, so I changed line 388 `sendBufferViaCPX(&txp, ImageOut, STREAM_W*STREAM_H);` to `sendBufferViaCPX(&txp, imgBuff0, CAM_WIDTH * CAM_HEIGHT);`, but I still see bounding boxes in `opencv-viewer.py`. How can I only stream the original images? Thank you.

**Comment by noahzn** [comment]
I still need help.

**Comment by krichardsson** [comment]
I do not know that much about this example, but the boxes are added by the GAP8. You could try to comment out the line that [runs the GAP8 code](https://github.com/bitcraze/aideck-gap8-examples/blob/8367be931fc041f48843de7f3ba8ecc250cd3469/examples/image_processing/FaceDetection/main.c#L378).

You could also take a look at the [image streamer example](https://github.com/bitcraze/aideck-gap8-examples/tree/master/examples/other/wifi-img-streamer)

**Reply by noahzn** [reply]
Hi, thank you for your reply. If I comment out the line you mentioned, the image viewer showed incorrect images like this:
![image](https://github.com/bitcraze/discussions/assets/8244850/7b2ac776-edf4-44c3-91ce-d3a0ec652f3c)

**Reply by noahzn** [reply, marked as answer]
The `task` is defined in `faceDet.c`. So I comment out [these lines](https://github.com/bitcraze/aideck-gap8-examples/blob/8367be931fc041f48843de7f3ba8ecc250cd3469/examples/image_processing/FaceDetection/faceDet.c#L451-L455). Now it works!
